build.gradle

```diff
@@ -147,6 +145,23 @@ project(':accurest-gradle-plugin') {
 		}
 	}
 
+	task createClasspathManifest {
+		def outputDir = file("$buildDir/$name")
+
+		inputs.files sourceSets.main.runtimeClasspath
+		outputs.dir outputDir
+
+		doLast {
+			outputDir.mkdirs()
+			file("$outputDir/plugin-classpath.txt").text = sourceSets.test.runtimeClasspath.join("\n")
+		}
+	}
+
+    // Add the classpath file to the test runtime classpath
+	dependencies {
+		testRuntime files(createClasspathManifest)
+	}
+
 	// Hack for reusing this in the plugin test
 	test.dependsOn(':accurest-core:install')
 	test.dependsOn(':accurest-messaging-root:accurest-messaging-core:install')
```

Comment:
WOW! Can you try removing those `test.dependsOn` and change them into:

```
compileOnly project(':accurest-messaging-root:accurest-messaging-integration')
```

and see if the build passes?